src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/OptionSet.txt

```diff
@@ -19,4 +19,7 @@ true
 4
 
 >> OtherOptionSet.OptionA <> TopOptionSetField
-Errors: Error 23-25: Incompatible types for comparison. These types can't be compared: OptionSetValue(OtherOptionSet), OptionSetValue(OptionSet).
\ No newline at end of file
+Errors: Error 23-25: Incompatible types for comparison. These types can't be compared: OptionSetValue(OtherOptionSet), OptionSetValue(OptionSet).
+
+>> IsBlank(Nested.InnerOtherOptionSet)
+false
```

Comment:
Can you also add some tests where it would return true, and tests for IsBlankOrError? Things like

>> IsBlank(If(1 < 0, OptionSet.Option1))
true

>> IsBlankOrError(OptionSet.Option1)
false

>> IsBlankOrError(If(1 < 0, OptionSet.Option1))
true

>> IsBlankOrError(If(1 / 0 < 2, OptionSet.Option1))
true

Reply:
IsBlankOrError isn't implemented yet, but yes, added the true case.